src/xcode/ENA/ENA/Source/Views/GradientView.swift

```diff
@@ -40,7 +40,7 @@ class GradientView: UIView {
 			UIColor(red: 0.663, green: 0.246, blue: 0.271, alpha: 1).cgColor
 		]
 		gradientLayer.locations = [0.12, 0.48, 0.81]
-		gradientLayer.startPoint = CGPoint(x: 0.05, y: 0.5)
-		gradientLayer.endPoint = CGPoint(x: 1.25, y: 0.5)
+		gradientLayer.startPoint = CGPoint(x: 0.0, y: 0.5)
+		gradientLayer.endPoint = CGPoint(x: 1.0, y: 1.0)
```

Comment:
Isn't there a problem on the checkin screen because it doesn't support your solution for the tilted gradient yet?

Reply:
I added a IBInspectable property to turn the tilting off, so it will still work with the checkin screen